src/react-skycons.js

```diff
@@ -44,6 +44,9 @@ const ReactSkycons = React.createClass({
    this.state.skycons.set(React.findDOMNode(this), Skycons[nextProps.icon]);
   },
 
+  componentWillUnmount: function componentWillUnmount() {
+    this.state.skycons.remove(React.findDOMNode(this));
```

Comment:
looks like `this.interval` might still be sticking around :| even after `.remove` since I do not see any call to `cancelInterval`